core/src/main/java/org/jclouds/http/internal/JavaUrlHttpCommandExecutorService.java

```diff
@@ -127,6 +128,11 @@ protected HttpResponse invoke(HttpURLConnection connection) throws IOException,
    }
 
    @Override
+   protected ListenableFuture<HttpResponse> invokeAsync(final HttpURLConnection nativeRequest) {
+      throw new UnsupportedOperationException("unsupported operation");
```

Comment:
Add some more information to let users know that what is not supported are the Async calls in general, and suggest using a different HTTP driver.